pkg/trivyk8s/trivyk8s_test.go

```diff
@@ -345,3 +351,112 @@ func TestInitResources(t *testing.T) {
 		})
 	}
 }
+
+type kubectlAction func() error
+
+func TestListSpecificArtifacts(t *testing.T) {
+	ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(10*time.Minute))
+	defer cancel()
+
+	k3sContainer, err := k3s.Run(ctx, "rancher/k3s:v1.27.1-k3s1")
+	require.NoError(t, err)
```

Comment:
I run into an error running these tests:

```shell
=== RUN   TestListSpecificArtifacts
--- FAIL: TestListSpecificArtifacts (0.00s)
panic: rootless Docker not found [recovered]
	panic: rootless Docker not found

```

What do I need to setup? I already have docker and Podman both.

Reply:
it's a bit strange error
I didn't install any additional tools. and the pipeline works correctly...

Reply:
I figured it out, we need to disable ryuk https://java.testcontainers.org/features/configuration/#disabling-ryuk